agent/runner.go

```diff
@@ -75,6 +75,16 @@ func (r *Runner) Run(ctx context.Context) error {
 		return nil
 	}
 
+	if len(work.Config.Stages) < 1 {
```

Comment:
I dont think this should affect the agent at all - it's a pipeline compiler thing

Reply:
You are right it should work without.